### ISSUE TITLE
Support Virtual Threads (Project Loom) in Native Image

### ISSUE THREAD

**Issue by christianwimmer** [oracle/graal#4920]
### TL;DR

OpenJDK project Loom adds virtual threads to JDK 19 (as a preview feature at first). Everything supported in OpenJDK should also be supported in GraalVM Native Image.

### Goals

Expand and finish the virtual thread support in GraalVM Native Image. Initial support had been added a while ago and is already available for the JDK 17 based version of GraalVM Native Image. But since then, the Project Loom API has changed considerably so the initial implementation needs to be updated. Also, we need to test that all places where JDK classes can yield virtual threads (locks, I/O, part events, ...) also yield properly on Native Image. Performance should be comparable to the HotSpot implementation.

Since virtual threads are a preview feature in OpenJDK, they will also be a preview feature for GraalVM Native Image.

### Non-Goals

We will not provide virtual thread support for the GraalVM releases based on JDK 11 and JDK 17.

**Comment by fniephaus** [comment]
Virtual Threads support has landed in GraalVM Native Image (see https://github.com/oracle/graal/pull/4802). :rocket:
